host/3.0/buster/amd64/powershell/powershell6-core-tools.Dockerfile

```diff
@@ -0,0 +1,100 @@
+
+FROM mcr.microsoft.com/dotnet/core/sdk:2.1
+
+# Avoid warnings by switching to noninteractive
+ENV DEBIAN_FRONTEND=noninteractive
+
+# This Dockerfile adds a non-root user with sudo access. Use the "remoteUser"
+# property in devcontainer.json to use it. On Linux, the container user's GID/UIDs
+# will be updated to match your local UID/GID (when using the dockerFile property).
+# See https://aka.ms/vscode-remote/containers/non-root-user for details.
+ARG USERNAME=vscode
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+
+ARG PS_VERSION=6.2.3
```

Comment:
I think we should install the latest LTS version (7.0.0). Please note that the version of PowerShell _supported by Azure Functions_ and the version of PowerShell _installed as an external tool_ are entirely independent. The Functions Core Tools include its own self-sufficient version, and the next release will include _both_ 6 and 7. The version installed here will _not_ be used by Azure Functions in any case: it will be used by commands and scripts invoked outside of the Functions context, so I think we should just install the latest LTS.

Reply:
@eamonoreilly @anirudhgarg What do you think?

Reply:
Hi @AnatoliB  That's make sense. Thank you for the review. I'd like to switch to 7.0.0 and test it. :) I need to learn more about PowerShell and the Language worker behavior!